source/Octopus.Server.Client/Model/Endpoints/AgentCommunicationStyleResource.cs

```diff
@@ -0,0 +1,14 @@
+using Octopus.TinyTypes;
+
+namespace Octopus.Client.Model.Endpoints;
+
+public class AgentCommunicationStyleResource : CaseInsensitiveStringTinyType
```

Comment:
To avoid confustion with the _other_ `CommunicationStyle` thing, should we maybe call this something slightly different (like `CommunicationMode` or somesuch)?

Reply:
I went with `AgentCommunicationMode/Resource`. Does that work for you?

Reply:
yep, perfect